modules/helpers.py

```diff
@@ -69,6 +69,13 @@ def target_creator(command_line_object):
 
         if root.tag.lower() == "nmaprun" and root.attrib.get('scanner') == 'nmap':
             print "Detected nmap xml file\n"
+
+            # command line provided ports
+            # in nmap logic, https ports must also be http 
+            http_ports += command_line_object.add_http_ports
+            http_ports += command_line_object.add_https_ports
```

Comment:
Quick question about this line (line 76), should it be removed?  It looks like you are adding https ports to http ports.